src/coreclr/src/inc/sbuffer.inl

```diff
@@ -183,6 +183,10 @@ inline SBuffer::~SBuffer()
         DeleteBuffer(m_buffer, m_allocation);
     }
 
+#ifdef _DEBUG
```

Comment:
Why is this needed?

Reply:
The initialization was missing from this constructor. The rest of the constructors of this class (above) are initializing it this way. GCC was complaining at its usage that this variable is initialized in Debug mode.